packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/tools/Root.spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { expect } from 'chai'
+import Root from './Root'
```

Comment:
Non-blocking and more of an FYI. I think with the new module spec (mjs) in javascript that if the export name and the filename do not match that it'll error. So, we should try to match the file names to the export name whenever possible. Maybe rename the file to `DrawingToolRoot`?